lib/jekyll/filters.rb

```diff
@@ -122,7 +122,8 @@ def normalize_whitespace(input)
     #
     # Returns the Integer word count.
     def number_of_words(input)
-      input.split.length
+      cjk_regex = %r(\p{Han}|\p{Katakana}|\p{Hiragana}|\p{Hangul})
```

Comment:
```suggestion
      cjk_regex = %r!\p{Han}|\p{Katakana}|\p{Hiragana}|\p{Hangul}!
```

This should make rubocop happy.